docs/install/quickstart-install.md

```diff
@@ -1,6 +1,7 @@
+--8<-- "quickstart-prereqs.md"
```

Comment:
This page appears to be titled "Before you begin" as this is the first H1 in this snippet. 

I think the title for this page should match the TOC, mentioning the word "quickstart".